### PR TITLE
Add comparison documentation for Kedge

### PR DESCRIPTION
**Pull request by cdrage**
This adds a comparison table against similar projects.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,17 @@ Go through our [file reference](http://kedgeproject.org/file-reference).
 
 Then go further with our pre-existing [examples](https://github.com/kedgeproject/kedge/tree/master/examples).
 
+__Comparison to similar projects:__
+
+|                                          | Kompose | Ksonnet | OpenShift templates | Helm | Kedge |
+|------------------------------------------|---------|---------|---------------------|------|-------|
+| Scaleable                                | -       | ✓       | ✓                   | ✓    | ✓     |
```
Comment (surajnarwade) [inline README.md]: `Scalable`

**Comment by cdrage** [comment]
Thanks @surajnarwade , updated :)

**Comment by containscafeine** [comment]
Umm, this shows Kedge as a competitor to the other tools, while it does not necessarily overlap with Helm, Templates, etc.

File: README.md [hunk 2]
```diff
@@ -50,6 +50,17 @@ Go through our [file reference](http://kedgeproject.org/file-reference).
...
 Then go further with our pre-existing [examples](https://github.com/kedgeproject/kedge/tree/master/examples).
 
+__Comparison to similar projects:__
+
+|                                          | Kompose | Ksonnet | OpenShift templates | Helm | Kedge |
+|------------------------------------------|---------|---------|---------------------|------|-------|
+| Scalable                                 | -       | ✓       | ✓                   | ✓    | ✓     |
+| Kubernetes compatible API and DSL Syntax | -       | -       | -                   | -    | ✓     |
+| Native to Kubernetes ecosystem           | -       | ✓       | -                   | ✓    | ✓     |
+| Integration to tools (IDEs)              | ✓       | ✓       | ✓                   | ✓    | ✓     |
```
Comment (surajnarwade) [inline README.md]: I am bit confused here for kompose

Comment (kadel) [reply README.md]: what @cdrage probably means is that basicaly every IDE that supporsts docker-compose has by definition support for kompose :-)

**Review by kadel — APPROVED** [review]
looks good. I would just consider explaining what Scalable means. For someone it might not be clear what is meant by that

**Review by concaf — CHANGES_REQUESTED** [review]
I'm nack for this :(
This gives a false impression that kedge is comparable or is in the same domain and space as Helm or OpenShift templates, and will add to the confusion.

**Comment by kadel** [comment]
> I'm nack for this :(
> This gives a false impression that kedge is comparable or is in the same domain and space as Helm or OpenShift templates, and will add to the confusion.

Almost everyone who sees Kedge for the first time asks quesion how Kedge compares (or how it is different) from those projects.

**We have answer this**.  
How do you suggest to explain it? It has to be in some short and easy to understand form.

**Comment by containscafeine** [comment]
Agree, we should address this, but not by comparing. We can have something like https://www.nomadproject.io/intro/vs/index.html

**Pull request closed without merging**